### PR TITLE
doc(Api) Added docstring to query function

### PR DESCRIPTION
**Pull request by jlunenburg**
And fixed spr bug

### CODE REVIEW

File: robot_skills/src/robot_skills/api.py [hunk 1]
```diff
@@ -12,10 +12,13 @@ def __init__(self, robot_name, tf_listener):
...
         self._client = Client(simple_action_client=client)
 
-
     def query(self, description, grammar, target, timeout=10):
         """
         Perform a HMI query, returns a HMIResult
+        :param description: text describing the query
+        :param grammar: string with the grammar to load
+        :param target: string identifying the target of the grammar to recognize
+        :param timeout: timeout in seconds (float)
```
Comment (LoyVanBeek) [inline robot_skills/src/robot_skills/api.py]: We don't have a grammar of the grammar-language, do we?

Comment (jlunenburg) [inline robot_skills/src/robot_skills/api.py]: I guess not. Would be good to have a description, but I guess this is not part of this merge request

**Comment by jlunenburg** [comment]
@Timple I thought all CI tests should pass but these still throw (unrelated?) errors, should these checks pass in your opinion?

**Comment by Timple** [comment]
I think we have to merge master back to have the correct travis file.

File: robot_skills/src/robot_skills/api.py (hunk 1 above)
Comment (LoyVanBeek) [inline robot_skills/src/robot_skills/api.py]: I know, I started writing some description in the grammar-parser repo

**Comment by jlunenburg** [comment]
Yeah, I see passes after the merge, in that case, should work next time we create a new branch ;)

**Pull request merged**